bcipy/oclm_language_model/demo/eeg_utils.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import division
```

Comment:
please delete this and consider using MNE or if random is okay the fake data  generated from the acquisition module: http://martinos.org/mne/stable/auto_examples/index.html#real-time-m-eeg-acquisition